build.go

```diff
@@ -56,6 +57,19 @@ func buildStaticSite(dest string) error {
 		}
 	})
 
+	// If we render 404 page
+	// Copy 404 page from dest/404/index.html to /dest/404.html
+	if in, err := os.Open(path.Join(dest, NOT_FOUND_PAGE, "index.html")); err == nil {
+		defer in.Close()
+		out, err := os.Create(path.Join(dest, "404.html"))
```

Comment:
do we need to move it to `404.html` or should we use the `NOT_FOUND_PAGE` value?

Reply:
github pages looking for 404.html or 404.md